<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceFile</class>
 <widget class="QWidget" name="SourceFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="margin">
    <number>12</number>
   </property>
   <item>
    <widget class="QsciScintilla" name="ui_editor">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ui_findFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Find</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ui_find"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Replace</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ui_replace"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="ui_matchCase">
          <property name="text">
           <string>Match Case</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ui_next">
          <property name="text">
           <string>Next</string>
          </property>
          <property name="icon">
           <iconset resource="../rc/rc.qrc">
            <normaloff>:/shortcuts/edit/icon_set/icons/arrow_redo.png</normaloff>:/shortcuts/edit/icon_set/icons/arrow_redo.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ui_replaceNext">
          <property name="text">
           <string>Replace and Next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ui_replaceAll">
          <property name="text">
           <string>Replace All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ui_closeFind">
          <property name="text">
           <string>Close</string>
          </property>
          <property name="icon">
           <iconset resource="../rc/rc.qrc">
            <normaloff>:/shortcuts/file/icon_set/icons/cross.png</normaloff>:/shortcuts/file/icon_set/icons/cross.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/file/icon_set/icons/disk.png</normaloff>:/shortcuts/file/icon_set/icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/bricks.png</normaloff>:/shortcuts/target/icon_set/icons/bricks.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/bullet_go.png</normaloff>:/shortcuts/target/icon_set/icons/bullet_go.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
  <action name="actionSimulate">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/application_go.png</normaloff>:/shortcuts/target/icon_set/icons/application_go.png</iconset>
   </property>
   <property name="text">
    <string>Simulate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/stop.png</normaloff>:/shortcuts/target/icon_set/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+.</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/ruby_blue.png</normaloff>:/shortcuts/target/icon_set/icons/ruby_blue.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/edit/icon_set/icons/page_white_copy.png</normaloff>:/shortcuts/edit/icon_set/icons/page_white_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/edit/icon_set/icons/page_white_paste.png</normaloff>:/shortcuts/edit/icon_set/icons/page_white_paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/edit/icon_set/icons/cut.png</normaloff>:/shortcuts/edit/icon_set/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/edit/icon_set/icons/arrow_undo.png</normaloff>:/shortcuts/edit/icon_set/icons/arrow_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/edit/icon_set/icons/arrow_redo.png</normaloff>:/shortcuts/edit/icon_set/icons/arrow_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/file/icon_set/icons/printer.png</normaloff>:/shortcuts/file/icon_set/icons/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionResetZoomLevel">
   <property name="text">
    <string>Reset Zoom Level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/edit/icon_set/icons/find.png</normaloff>:/shortcuts/edit/icon_set/icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionChangeTarget">
   <property name="text">
    <string>Change Target</string>
   </property>
  </action>
  <action name="actionChoosePort">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/link.png</normaloff>:/shortcuts/target/icon_set/icons/link.png</iconset>
   </property>
   <property name="text">
    <string>Choose Port</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/bug.png</normaloff>:/shortcuts/target/icon_set/icons/bug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionToggleBreakpoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/sourcefile/icon_set/icons/bullet_blue.png</normaloff>:/sourcefile/icon_set/icons/bullet_blue.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Breakpoint</string>
   </property>
   <property name="toolTip">
    <string>Toggle Breakpoint</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionClearBreakpoints">
   <property name="text">
    <string>Clear Breakpoints</string>
   </property>
  </action>
  <action name="actionIndentAll">
   <property name="text">
    <string>Indent All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUi">
   <property name="icon">
    <iconset resource="../rc/rc.qrc">
     <normaloff>:/shortcuts/target/icon_set/icons/bricks.png</normaloff>:/shortcuts/target/icon_set/icons/bricks.png</iconset>
   </property>
   <property name="text">
    <string>Ui</string>
   </property>
  </action>
  <action name="actionRequestFile">
   <property name="text">
    <string>Request File</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ui_find</sender>
   <signal>returnPressed()</signal>
   <receiver>ui_next</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_closeFind</sender>
   <signal>clicked()</signal>
   <receiver>ui_findFrame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFind</sender>
   <signal>triggered()</signal>
   <receiver>ui_find</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
